src/net_processing.cpp

```diff
@@ -172,6 +173,8 @@ static constexpr double MAX_ADDR_RATE_PER_SECOND{0.1};
  *  based increments won't go above this, but the MAX_ADDR_TO_SEND increment following GETADDR
  *  is exempt from this limit). */
 static constexpr size_t MAX_ADDR_PROCESSING_TOKEN_BUCKET{MAX_ADDR_TO_SEND};
+/** Supported transaction reconciliation protocol version */
+static constexpr uint32_t RECON_VERSION = 1;
```

Comment:
in d53b4bc0360760737b250b81e82c4d8e6e6b1144:

nit
```suggestion
static constexpr uint32_t RECON_VERSION{1};
```